### PR TITLE
Review Setup

### PR DESCRIPTION
**Pull request by vinnywalker96**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>tech.aluve</groupId>
+    <artifactId>assessment3</artifactId>
```
Comment (sibo-t) [inline pom.xml]: please change this to aluve_calendar_qa

File: src/src/test/java/api/cucumber/RunCucumberIT.java
Comment (sibo-t) [inline src/src/test/java/api/cucumber/RunCucumberIT.java:1]: Please remove this entire directory, because "src" already exists

**Review by sibo-t — CHANGES_REQUESTED** [review]

**Review by spheleleMakau — COMMENTED** [review]
On the pom.xml looks like you are building assessement3 (line 8)
Run CucumberIT file includes tags as "book"

**Comment by vinnywalker96** [comment]
Cannot re-open PR

**Comment by vinnywalker96** [comment]
Review

**Pull request closed without merging**